hc/accounts/tests/test_profile.py

```diff
@@ -42,9 +52,16 @@ def test_it_adds_team_member(self):
 
         ### Assert the existence of the member emails
 
-        self.assertTrue("frank@example.org" in member_emails)
+        self.assertTrue("ben@example.org" in member_emails)
```

Comment:
***CodeQuaility: Nitpick***
This code should be removed as it's just a smoke test